packaging/hudi-presto-bundle/pom.xml

```diff
@@ -158,11 +158,66 @@
       <groupId>org.apache.hudi</groupId>
       <artifactId>hudi-common</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-server</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-client</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
+
     <dependency>
       <groupId>org.apache.hudi</groupId>
       <artifactId>hudi-hadoop-mr-bundle</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-server</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-client</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <!-- https://mvnrepository.com/artifact/org.apache.hbase/hbase-shaded-client -->
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-shaded-client</artifactId>
+      <version>${hbase.version}</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-shaded-server</artifactId>
```

Comment:
Can we also introduce this dependency in hudi-common?

Reply:
emmmm, changing zhe hudi-common dependency is relatively high risk. Because not only metadata table's Hfile using hbase dependency but also hbase index or something else.
Also there seems no conflict between spark and flink bundle.

So that, the changes in presto bundle is much more safety and just solve the conflicts i meet between hbase and presto.

Reply:
+1 
@zhangyue19921010  If we add to hudi-common then it should be sufficient. hudi-common is a dependency in hudi-presto-bundle. It also makes sense from the perpective of integrating with Presto. That way we will only need to add hudi-common dependency in presto-hive module (which should be lighter than adding hudi-presto-bundle).